src/main/java/frc/robot/commands/AutonBalance.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.PIDCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.subsystems.DriveSubsystem;
+
+public class AutonBalance extends SequentialCommandGroup {
+
+  private class Balance extends PIDCommand {
```

Comment:
See if you can make this an inline PIDCommand.

Reply:
Rewrote it with composition, I don't believe it will behave differently. However, with writing it inline, I can only get the command to stop with an `until` call since PIDCommand does not implement the `isFinished` method on its own.